### PR TITLE
Kogito 1.7.0.Final + OptaPlanner 8.7.0.Final

### PR DESCRIPTION
**Pull request by radtriste**

### CODE REVIEW

**Comment by radtriste** [comment]
@gsmet can you approve for the workflow to run ?

**Comment by radtriste** [comment]
cc @geoand

File: pom.xml [hunk 1]
```diff
@@ -79,6 +79,25 @@
...
+            <url>https://repository.jboss.org/nexus/content/groups/kogito-public/</url>
+        </repository>
+        <repository>
+            <snapshots>
+            <enabled>false</enabled>
+            </snapshots>
+            <id>central</id>
+            <name>Maven Central</name>
+            <url>https://repo.maven.apache.org/maven2</url>
+        </repository>
```
Comment (geoand) [inline pom.xml:98]: Is this really needed?

Comment (radtriste) [reply pom.xml:98]: @evacchi ? :D

Comment (gsmet) [reply pom.xml:98]: If they have been properly released to Maven Central, this is not necessary.

Comment (radtriste) [reply pom.xml:98]: they are not yet released to Maven Central, That is why we have also https://github.com/quarkusio/quarkus-platform/pull/239/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R83-R90

Comment (radtriste) [reply pom.xml:98]: before releasing we check that the platform PR is ok :)

Comment (radtriste) [reply pom.xml:98]: @gsmet @geoand Could we at least launch the check ?
anyway this will be removed before merging the PR ...

Comment (radtriste) [reply pom.xml]: removed

Comment (gsmet) [reply pom.xml]: ah yes sorry about not launching the check, not used to this new GH feature. As far as I can see CI is green now.

**Comment by gsmet** [comment]
@radtriste should we mark it ready for review now?

**Comment by radtriste** [comment]
@gsmet one thing I notice in the check:
```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Quarkus Platform - Parent 999-SNAPSHOT:
[INFO] 
[INFO] Quarkus Platform - Quarkus Universe - Quarkus Platform BOM SUCCESS [  0.183 s]
[INFO] Quarkus Platform - Parent .......................... SUCCESS [  0.754 s]
[INFO] Quarkus Platform - Quarkus Universe - Parent ....... SUCCESS [  0.035 s]
[INFO] Quarkus Platform - Quarkus Universe - Quarkus Platform Descriptor SUCCESS [ 50.061 s]
[INFO] Quarkus Platform - Quarkus Universe - Quarkus Platform Properties SUCCESS [  0.028 s]
[INFO] Quarkus Platform - Core - Quarkus Platform BOM ..... SUCCESS [  0.008 s]
[INFO] Quarkus Platform - Core - Parent ................... SUCCESS [  0.015 s]
[INFO] Quarkus Platform - Core - Quarkus Platform Descriptor SUCCESS [  0.658 s]
[INFO] Quarkus Platform - Core - Quarkus Platform Properties SUCCESS [  0.014 s]
[INFO] Quarkus Platform - Hazelcast - Quarkus Platform BOM  SUCCESS [  0.003 s]
[INFO] Quarkus Platform - Hazelcast - Parent .............. SUCCESS [  0.008 s]
[INFO] Quarkus Platform - Hazelcast - Integration Tests - Parent SUCCESS [  0.006 s]
[INFO] Quarkus Platform - Hazelcast - Integration Tests - quarkus-hazelcast-client-integration-tests SUCCESS [ 16.892 s]
[INFO] Quarkus Platform - Hazelcast - Quarkus Platform Descriptor SUCCESS [  0.092 s]
[INFO] Quarkus Platform - Hazelcast - Quarkus Platform Properties SUCCESS [  0.025 s]
[INFO] Quarkus Platform - QpidJMS - Quarkus Platform BOM .. SUCCESS [  0.016 s]
[INFO] Quarkus Platform - QpidJMS - Parent ................ SUCCESS [  0.010 s]
[INFO] Quarkus Platform - QpidJMS - Integration Tests - Parent SUCCESS [  0.020 s]
[INFO] Quarkus Platform - QpidJMS - Integration Tests - quarkus-qpid-jms-integration-tests SUCCESS [  9.967 s]
[INFO] Quarkus Platform - QpidJMS - Quarkus Platform Descriptor SUCCESS [  0.030 s]
[INFO] Quarkus Platform - QpidJMS - Quarkus Platform Properties SUCCESS [  0.020 s]
[INFO] Quarkus Platform - Blaze-Persistence - Quarkus Platform BOM SUCCESS [  0.003 s]
[INFO] Quarkus Platform - Blaze-Persistence - Parent ...... SUCCESS [  0.010 s]
[INFO] Quarkus Platform - Blaze-Persistence - Integration Tests - Parent SUCCESS [  0.008 s]
[INFO] Quarkus Platform - Blaze-Persistence - Integration Tests - blaze-persistence-examples-quarkus-testsuite-base SUCCESS [ 16.933 s]
[INFO] Quarkus Platform - Blaze-Persistence - Integration Tests - blaze-persistence-examples-quarkus-testsuite-native-h2 SUCCESS [  0.066 s]
[INFO] Quarkus Platform - Blaze-Persistence - Quarkus Platform Descriptor SUCCESS [  0.046 s]
[INFO] Quarkus Platform - Blaze-Persistence - Quarkus Platform Properties SUCCESS [  0.043 s]
[INFO] Quarkus Platform - Quarkus Maven Plugin ............ SUCCESS [  1.450 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:40 min
[INFO] Finished at: 2021-06-09T09:22:13Z
[INFO] ------------------------------------------------------------------------
```
Kogito/Optaplanner does not seem to be checked

**Comment by radtriste** [comment]
and I would say that Kogito artifacts are not yet in Maven Central ... (I just released them this morning from JBoss Nexus)

**Comment by radtriste** [comment]
See also https://repo1.maven.org/maven2/org/kie/kogito/kogito-build-parent/

**Comment by gsmet** [comment]
Ah yes, they are not tested because they didn't work with Quarkus 2.0. Are they working now? If so we can enable them.

**Comment by gsmet** [comment]
This would need to be set to `true` if your upgrade works with 2.0 (and once the artifacts are on Central): https://github.com/quarkusio/quarkus-platform/blob/main/pom.xml#L212

**Comment by radtriste** [comment]
They do not :S 
Quarkus 2.0 integration is still under dev

**Comment by radtriste** [comment]
@gsmet I tell you once artifacts are on Maven Central

**Comment by radtriste** [comment]
@gsmet all good for merging: https://repo1.maven.org/maven2/org/kie/kogito/kogito-build-parent/1.7.0.Final/

**Comment by radtriste** [comment]
@gsmet @geoand could we merge that one ?

**Comment by geoand** [comment]
Thanks!

**Pull request merged**